planetwars/client/webpack/app/app.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
```

Comment:
Is this file necessary? What does it do?

Reply:
No idea where that came from, sorry.